### PR TITLE
feat: switch from dna to spectrum-tokens

### PR DESCRIPTION
**Pull request by GarthDB**
## Description
This uses the public spectrum-tokens npm package instead of the internal-only DNA package.

It also contains the latest vars from DNA.

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

**Comment by jianliao** [comment]
@GarthDB seems like `@adobe/spectrum-tokens` depends on `@spectrum/kulcon` which is from our private artifactory. It will fail the build if we don't put `@adobe/spectrum-tokens` into optionalDependency. Could you check if you can remove the dependency for `@spectrum/kulcon`? then republish `@adobe/spectrum-tokens`.

**Comment by GarthDB** [comment]
@jianliao tokens should only have devDependencies - it is only the json flies from dna

**Comment by jianliao** [comment]
Here are the dependency definitions extracted from [@adobe/spectrum-tokens/package.json](https://unpkg.com/@adobe/spectrum-tokens@6.0.0/package.json).

```
  "dependencies": {
    "@spectrum/kulcon": "^1.x",
    "bluebird": "^3.x",
    "color-alpha": "^1.0.3",
    "deep-extend": "^0.6.0",
    "fs-extra": "^4.x",
    "globby": "^6.x",
    "node-es-module-loader": "^0.3.8",
    "xtend": "^4.x"
  },
  "devDependencies": {
    "@spectrum/dna-npm-release": "^2.1.0",
    "@spectrum/eslint-config-dna": "^1.x",
    "babel-eslint": "^10.0.2",
    "chai": "^4.x",
    "child-process-es6-promise": "^1.2.0",
    "eslint": "^4.12.1",
    "esm": "^3.2.25",
    "mocha": "^5.2.0",
    "npm-run-all": "^4.1.5",
    "onchange": "^6.0.0",
    "rimraf": "^2.x",
    "sinon": "^4.x"
  }
```
The spectrum-css build failed on those private package dependencies above.

**Comment by GarthDB** [comment]
Yeah, I’ll change all of the deps to devDependencies

**Comment by jianliao** [comment]
hmm...I think npm/yarn will still try to install devDeps. But I am not sure.

**Comment by jianliao** [comment]
@GarthDB, please check the above VR result. Everything looks good to me.

File: components/vars/css/components/spectrum-banner.css [hunk 1]
```diff
@@ -1,11 +1,130 @@
...
+  --spectrum-banner-16-text-color: a;
+  --spectrum-banner-17-text-color: t;
+  --spectrum-banner-18-text-color: i;
+  --spectrum-banner-19-text-color: c;
+  --spectrum-banner-20-text-color: -;
+  --spectrum-banner-21-text-color: w;
+  --spectrum-banner-22-text-color: h;
+  --spectrum-banner-23-text-color: i;
+  --spectrum-banner-24-text-color: t;
+  --spectrum-banner-25-text-color: e;
```
Comment (jianliao) [inline components/vars/css/components/spectrum-banner.css:27]: Seems like this is wrong.

**Pull request merged**